homeassistant/components/lyric/sensor.py

```diff
@@ -0,0 +1,131 @@
+                device, location, hass, 'outdoorTemperature',
+                'Temperature Outside',
+                hass.config.units.temperature_unit, 'mdi:thermometer',
+                DEVICE_CLASS_TEMPERATURE))
+        if device.displayedOutdoorHumidity:
+            devices.append(LyricSensor(
+                device, location, hass, 'displayedOutdoorHumidity',
+                'Humidity Outside', '%', 'mdi:water-percent',
+                DEVICE_CLASS_HUMIDITY))
+        if device.nextPeriodTime:
+            devices.append(LyricSensor(
+                device, location, hass, 'nextPeriodTime',
+                'Next Period Time', '', 'mdi:clock',
+                DEVICE_CLASS_TIMESTAMP))
+        if device.thermostatSetpointStatus:
+            devices.append(LyricSensor(
+                device, location, hass, 'thermostatSetpointStatus',
+                'Status', '', 'mdi:thermostat', None))
+
+    async_add_entities(devices, True)
+
+
+class LyricSensor(LyricDeviceEntity):
+    """Representation of a Lyric thermostat."""
+
+    def __init__(self, device, location, hass, key, key_name,
+                 unit_of_measurement=None, icon=None,
+                 device_class=None) -> None:
+        """Initialize the sensor."""
+        unique_id = '{}_{}'.format(
+            device.macID, key_name.replace(" ", "_"))
+        self._unit_of_measurement = unit_of_measurement
+        self._state = None
+        self._available = False
+        self.key = key
+
+        name = '{} {}'.format(device.name, key_name)
+
+        super().__init__(device, location, unique_id, name, icon, device_class)
+
+    @property
+    def state(self):
+        """Return the state of the entity."""
+        return self._state
+
+    @property
+    def unit_of_measurement(self) -> str:
+        """Return the unit of measurement."""
+        return self._unit_of_measurement
+
+    async def _lyric_update(self) -> None:
+        """Get values from lyric."""
+        if self.device:
+            if self.key == 'thermostatSetpointStatus':
+                status = getattr(self.device, self.key)
+                if status == 'NoHold':
+                    self._state = 'Following Schedule'
+                elif status == 'HoldUntil':
+                    self._state = 'Held until {}'.format(
+                        self.device.nextPeriodTime[:-3])
+                elif status == 'PermanentHold':
+                    self._state = 'Held Permanently'
+                elif status == 'VacationHold':
+                    self._state = 'Holiday'
+            else:
+                state = getattr(self.device, self.key)
+                if self._device_class == DEVICE_CLASS_TIMESTAMP:
+                    time = dt_util.parse_time(state)
+                    now = dt_util.utcnow()
+                    if time <= now.time():
+                        now = now + timedelta(days=1)
+                    state = dt_util.as_local(dt_util.as_utc(
```

Comment:
The state should be utc iso formatted time.

Reply:
If I remove the `dt_util.as_local` the time is now UTC and the entity shows the correct time diff, however in the more info dialog shows this in UTC format instead of +1 which is where it should be where I am. Is this a bug in the frontend to report or am I formatting the state incorrectly?

![image](https://user-images.githubusercontent.com/28114703/63510596-dc534180-c4d6-11e9-8530-ff2a6775d78a.png)

![image](https://user-images.githubusercontent.com/28114703/63510577-cfcee900-c4d6-11e9-9de8-363a55efcb05.png)